aiven/resource_vpc_peering_connection.go

```diff
@@ -191,10 +191,29 @@ func parsePeeringVPCId(resourceID string) (string, string, string, string, *stri
 }
 
 func resourceVPCPeeringConnectionRead(d *schema.ResourceData, m interface{}) error {
+	var pc *aiven.VPCPeeringConnection
 	client := m.(*aiven.Client)
 
 	projectName, vpcID, peerCloudAccount, peerVPC, peerRegion := parsePeeringVPCId(d.Id())
-	pc, err := client.VPCPeeringConnections.GetVPCPeering(projectName, vpcID, peerCloudAccount, peerVPC, peerRegion)
+	isAzure, err := isAzureVPCPeeringConnection(d, client)
+	if err != nil {
+		return err
+	}
+
+	if isAzure {
+		if peerResourceGroup, ok := d.GetOk("peer_resource_group"); ok {
+			pc, err = client.VPCPeeringConnections.GetVPCPeeringWithResourceGroup(
+				projectName, vpcID, peerCloudAccount, peerVPC, peerRegion, peerResourceGroup.(string))
+			if err != nil {
+				return err
+			}
+		} else {
+			return errors.New("cannot get an Azure VPC peering connection without `peer_resource_group`")
+		}
+	}
+
+	pc, err = client.VPCPeeringConnections.GetVPCPeering(
```

Comment:
This is now executed also for Azure VPC peering connections.

Reply:
ups, indeed, fixed